### DISCUSSION TITLE
custom form-field reactive

### DISCUSSION THREAD

**Discussion by photojo** [filamentphp/filament#13267] [Help] [answered]
### Package

Form builder

### Package Version

v3.1

### How can we help you?

Can I use 
->reactive() and ->afterStateUpdated(..) on custom form-fields too?

**Comment by leandrocfe** [comment]
You should use `$applyStateBindingModifiers('wire:model')`
https://filamentphp.com/docs/3.x/forms/fields/custom#obeying-state-binding-modifiers

**Comment by photojo** [comment, marked as answer]
Works fine! Thank you!
